sirepo/auth_db.py

```diff
@@ -324,8 +325,16 @@ class UserRoleInvite(UserDbBase):
 
         @classmethod
         def get_moderation_request_rows(cls):
+            import sirepo.auth
```

Comment:
Don't qualify imports inside of function. When you import like this the module is added to sys.modules and then ins't removed when we exit the function which it should be (a bug of python). So import like `from sirepo import auth`

Reply:
6a18786a1c0ea80d306b558a3c9d83ef8d1df2d8